lib/seahorn/BvOpSem2.cc

```diff
@@ -157,6 +176,10 @@ bool isShadowMem(const Value &V, const Value **out) {
   return res;
 }
 
+bool isUnifiedAssume(const CallInst &CI) {
```

Comment:
this should be in the same file that sets the mark so that the file can control how the marking is done.

Reply:
easier if this works on arbitrary instruction not just `CallInst`.